docs/docs/reference/shards/lisp/misc.md

```diff
@@ -33,44 +33,92 @@ This is useful for cases where you need multiple shards (to transform your data)
 
 ## apply
 
-```clojure linenums="1"
+```
+(apply f args) (apply f x args) (apply f x y args) (apply f x y z args)(apply f a b c d & args)
+```
+
+Applies fn f to the argument list formed by prepending intervening arguments to args.
 
+```clojure linenums="1"
+(apply str ["cherry " "banana " "apple"])
+;;=> "cherry banana apple"
 ```
 
 ## atom
 
-```clojure linenums="1"
+```
+(atom x)
+```
 
+  Creates and returns an Atom with an initial value of x and zero or more options (in any order).
+
+```clojure linenums="1"
+(atom {})
 ```
 
 ## deref
 
-```clojure linenums="1"
+```
+(deref ref)
+```
+
+Returns the most-recently-committed value of ref. When applied to an atom, returns its current state.
 
+```clojure linenums="1"
+(deref (atom [1 2]))
+;;=> [1 2]
 ```
 
 ## fn?
 
-```clojure linenums="1"
+```
+(fn? f)
+```
 
+Return true if f is a function.
+
+```clojure linenums="1"
+(fn? String.Join)
+;;=> true
 ```
 
 ## hex
 
-```clojure linenums="1"
+```
+(hex number)
+```
+
+Returns an hexidecimal representation for input number.
 
+```clojure linenums="1"
+(hex 2)
+;;=> "0x0000000000000002"
 ```
 
 ## keyword
 
-```clojure linenums="1"
+```
+(keyword s)
+```
 
+Returns a keyword with given s string. Do not use `:` in the keyword strings, it will be added automatically.
+
+```clojure linenums="1"
+(keyword "foo/bar")
+;;=> :foo/bar
 ```
 
 ## macro?
 
-```clojure linenums="1"
+```
+(macro? form)
+```
 
+Checks if form is a macro.
```

Comment:
What's a form?

Reply:
> A form is a list whose first element is a symbol that names an operator. If the first element names a function, then the form is a functional form.

It excludes data data.

Reply:
Although this doesn't make sense in Mal.

Reply:
Now I mention value. Ima wa dou?

Reply:
I think mal also uses the terminology 'form', I'm just no sure what it's definition is

Reply:
It's just a lispy umbrella for a lot of possible stuff under `()`. Like saying "object".